cfme/automate/explorer/method.py

```diff
@@ -378,7 +378,10 @@ def create(
             return None
         else:
             add_page.add_button.click()
-            msg = 'Automate Method "{}" was added'.format(name)
+            if self.appliance.version < '5.9':
```

Comment:
This will fail other cases ex.cfme/tests/automate/test_vmware_methods.py/test_vmware_vimapi_hotadd_disk.py As I understood there is inconsistance in flash message while method creation with different conditions. @nzr-zts worked on fix for test_vmware_methods tests

Reply:
@quarckster Let's just assert no errors here.